fetch.bs

```diff
@@ -6710,6 +6752,417 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+
+   <li>
+    <p>If <var>requestClientDocument</var>'s <a for=Document>origin</a> is <a>same origin</a> with
+    <var>otherDocument</var>'s <a for=Document>origin</a>, then:
+    <ol>
+     <li>
+      <p>If <var>otherContainer</var> is null, then:
+
+      <p class=note>Accumulate the <a for=/>top-level traversable</a>'s initial quota.
+
+      <ol>
+       <li><p>If <var>otherDocument</var> is not <a>allowed to use</a> the
+       <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch}}", then return 0.
+
+       <li><p>Assert: <var>quota</var> is 0.
+
+       <li>
+        <p>Set <var>quota</var> be 640 kibibytes.
+        <p class="note allow-2119">640kb should be enough for everyone.
+
+       <li><p>If <var>otherDocument</var> is <a>allowed to use</a> the
+       <a>policy-controlled feature</a> "{{PermissionsPolicy/deferred-fetch-minimal}}", then
+       decrement <var>quota</var> by <a>max containers with minimal quota</a>, multiplied by
+       <a for="reserved deferred-fetch quota">minimal quota</a>.
+      </ol>
+
+     <li>
+      <p>Otherwise, if any of the following conditions is true:
+
+      <ul class=brief>
+       <li><p><var>otherContainer</var>'s <a>reserved deferred-fetch quota</a> is
+       <a for="reserved deferred-fetch quota">normal quota</a>, and <var>otherDocument</var> is
+       <a>allowed to use</a> the <a>policy-controlled feature</a>
+       "{{PermissionsPolicy/deferred-fetch}}"
+
+       <li><p><var>otherContainer</var>'s <a>reserved deferred-fetch quota</a> is
+       <a for="reserved deferred-fetch quota">minimal quota</a>, and <var>otherDocument</var> is
+       <a>allowed to use</a> the <a>policy-controlled feature</a>
+       "{{PermissionsPolicy/deferred-fetch-minimal}}"
+      </ul>
+
+      <p>then increment <var>quota</var> by <var>otherContainer</var>'s
+      <a>reserved deferred-fetch quota</a>.
+      <p class=note>Accumulate quota granted by parent documents.
+
+     <li>
+      <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> of
+      <var>otherDocument</var>'s <a for=fetch>fetch group</a>'s
+      <a for="fetch group">deferred fetch records</a>:</p>
+
+      <p class=note>Account for quota on deferred fetches performed by <a>same origin</a> clients.
+
+      <ol>
+       <li><p>Let <var>requestLength</var> be the <a>total request length</a> of
+       <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>.
+
+       <li><p>Decrement <var>quota</var> by <var>requestLength</var>.
+
+       <li><p>If <var>deferredRecord</var>'s <a for="deferred fetch record">request</a>'s
+       <a for=request>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>origin</var>,
+       then decrement <var>quotaForOrigin</var> by <var>requestLength</var>.
+      </ol>
+    </ol>
+
+   <li>
+    <p>If <var>otherDocument</var>'s <a>container document</a> is a {{Document}} whose
```

Comment:
```
If otherDocument’s container document is a Document whose origin is same origin with requestClientDocument’s origin,
```

If `otherDocument` is the root document, `otherDocument’s container document` will be null right?